seacrowd/sea_datasets/indoner_tourism/indoner_tourism.py

```diff
@@ -0,0 +1,196 @@
+    issn = {2621-9336}, pages = {30--35},   doi = {10.26418/jlk.v6i1.89},
+    url = {https://inacl.id/journal/index.php/jlk/article/view/89}
+}
+"""
+
+_DATASETNAME = "indoner_tourism"
+
+_DESCRIPTION = """\
+This dataset is designed for named entity recognition (NER) tasks in the Bahasa Indonesia tourism domain. It contains labeled sequences of named entities, including locations, facilities, and tourism-related entities. The dataset is annotated with the following entity types:
+
+    O (0)    : Non-entity or other words not falling into the specified categories.
+    B-WIS (1): Beginning of a tourism-related entity.
+    I-WIS (2): Continuation of a tourism-related entity.
+    B-LOC (3): Beginning of a location entity.
+    I-LOC (4): Continuation of a location entity.
+    B-FAS (5): Beginning of a facility entity.
+    I-FAS (6): Continuation of a facility entity.
+"""
+
+_HOMEPAGE = "https://github.com/fathanick/IndoNER-Tourism/tree/main"
+
+_LANGUAGES = ['ind']  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+
+_LICENSE = Licenses.AFL_3_0.value 
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: "https://raw.githubusercontent.com/fathanick/IndoNER-Tourism/main/ner_data.tsv",
+}
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class IndoNERTourism(datasets.GeneratorBasedBuilder):
+    """\
+This dataset is designed for named entity recognition (NER) tasks in the Bahasa Indonesia tourism domain. It contains labeled sequences of named entities, including locations, facilities, and tourism-related entities. The dataset is annotated with the following entity types:
+
+    O (0)    : Non-entity or other words not falling into the specified categories.
+    B-WIS (1): Beginning of a tourism-related entity.
+    I-WIS (2): Continuation of a tourism-related entity.
+    B-LOC (3): Beginning of a location entity.
+    I-LOC (4): Continuation of a location entity.
+    B-FAS (5): Beginning of a facility entity.
+    I-FAS (6): Continuation of a facility entity.
+"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="indoner_tourism_source",
```

Comment:
Let's use `_DATASETNAME` here (and also in the `subset_id`)

Reply:
Implemented!